action/protocol/rewarding/protocol.go

```diff
@@ -68,6 +68,16 @@ func NewProtocol(cfg genesis.Rewarding) *Protocol {
 	}
 }
 
+// ProtocolAddr returns the address generated from protocol id
+func ProtocolAddr() address.Address {
```

Comment:
create a common func `hashStringToAddress()` under `/action/protocol`
and here
```
return hashStringToAddress(protocolID)
```

Reply:
I don't think exporting protocolID is better